### ISSUE TITLE
Dynazoom limit (Y scale) is always 0-1

### ISSUE THREAD

**Issue by terba** [munin-monitoring/munin#815]
Hello,

On Centos 7 after upgrading from 2.0.29 to 2.0.30, dynazoom is always displaying the graphs with an Y scale from 0 to 1. If I enter values into the Limit low/high inputs and press Submit query, it displays ok.

Thanks!

**Comment by terba** [comment]
I solved it by modifing the following lines in munin-cgi-graph from **if defined** to **if ne ""** like this:
```  
    my $upper_limit = CGI::param("upper_limit");
    push @params, "--upper_limit", $upper_limit if $upper_limit ne "";

    my $lower_limit = CGI::param("lower_limit");
    push @params, "--lower_limit", $lower_limit if $lower_limit ne "";
```

**Comment by fedelebrero** [comment]
Same problem on CentOS 6.x after upgrading to 2.0.30.

This solution also applies to CentOS 6.x

**Comment by steveschnepp** [comment]
Could you try instead the fix in 6373554b1cc8bee886947cee598e86d1d9ea1e4a that is in 2.0.33 ?

**Comment by terba** [comment]
@steveschnepp It works, thanks!
